core/templates/dev/head/pages/topics_and_skills_dashboard/merge_skill_modal_directive.html

```diff
@@ -0,0 +1,25 @@
+<div class="modal-header">
+  <h3>Select the skill to merge into</h3>
+</div>
+
+<div class="modal-body">
+  <p>
+    Merging into skill: <[selectedSkill.description]>
+  </p>
+  <div class="skills">
+    <skills-list skill-summaries="skillSummaries" in-modal="true"
+                 selected-skill="selectedSkill">
+    </skills-list>
+  </div>
+</div>
+
+<div class="modal-footer">
+  <button class="btn btn-default" ng-click="cancel()">Cancel</button>
+  <button class="btn btn-success" ng-click="done()">Save</button>
+</div>
+
+<style>
+  .topics-table {
```

Comment:
Where is this used?

Reply:
Was used previously but I made some changes where we dont need it and forgot to remove the style sorry